dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/oracle/AbstractOracleConnector.java

```diff
@@ -184,7 +184,7 @@ Properties buildProperties(@Nonnull ConnectorArguments arguments) {
   // dbc:oracle:thin://<host>:<port>/<service>
   // jdbc:oracle:thin:<host>:<port>:<SID>
   // jdbc:oracle:thin:<TNSName> (from 10.2.0.1.0)
-  private String buildUrl(ConnectorArguments arguments) {
+  static String buildUrl(ConnectorArguments arguments) {
```

Comment:
Optional: If it makes sense to change the method to `static`, then I would recommend extracting the logic to a separate class (e.g. `OracleUrlBuilder`).

Reply:
I agree with this idea, in general I think that AbstractOracleConnector could be slimmed down or even removed eventually. However I'd prefer to do this later and limit this PR to what's required for testability.